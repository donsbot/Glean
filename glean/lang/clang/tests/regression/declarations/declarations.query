query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
