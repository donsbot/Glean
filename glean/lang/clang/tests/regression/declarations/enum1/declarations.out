[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 1,
      "lineEnd": 12,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 3,
      "lineEnd": 10,
      "columnEnd": 25
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 3,
      "lineEnd": 11,
      "columnEnd": 39
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 14,
      "columnBegin": 1,
      "lineEnd": 14,
      "columnEnd": 26
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 15,
      "columnBegin": 1,
      "lineEnd": 15,
      "columnEnd": 40
    }
  }
]